README.md

```diff
@@ -209,6 +210,17 @@ below shows its use:
 <b>Check Output</b>: {{.Check.Output}}
 ```
 
+## Debugging
+
+It can be helpful to run from the command line to debug issues such as authentication. For this you will need an event. The command below assumes you are running this on a sensu backend.
+
+```
+sensuctl event info `hostname -f` keepalive --format=json | \
```

Comment:
Probably need to explain this is an example and what that `hostname -f` is doing.  That's a nice shortcut for some cases, where entity names match fqdn and you know the host you are testing on is running an agent.